app/ts/EffectsTable/EffectsTableCriteriaRows/EffectsTableDataSourceRow/ValueCell/EffectValueCell/EffectValueCellService.ts

```diff
@@ -147,3 +147,14 @@ function getValueFromScales(scale: IScale, usePercentage: boolean): string {
     return 'No data entered';
   }
 }
+
+export function normalizeInputValue(
+  value: string,
+  unitType: UnitOfMeasurementType
+): number {
+  if (!value || unitType != 'percentage') {
+    return Number.parseFloat(value);
+  } else {
+    return Number.parseFloat(value) / 100;
```

Comment:
Where is this function being used? I thought we only did *100 in display cells. Is this function in the wrong file?

Reply:
wrong file, yes. This is the division case used during manual input.